src/rules.jl

```diff
@@ -518,6 +530,42 @@ function lrp!(Rᵏ, rule::GeneralizedGammaRule, layer, modified_layers, aᵏ, R
     @. Rᵏ = aᵏ⁺ * (cˡ⁺ + cʳ⁻) + aᵏ⁻ * (cˡ⁻ + cʳ⁺)
 end
 
+"""
+    LayerNormRule(affine_rule=ZeroRule())
+
+LayerNormRule rule.
+
+# Optional arguments
+- `affine_rule`: Rule applied to affine transformation, defaults to `ZeroRule()`.
+
+# References
+- 
+"""
+struct LayerNormRule{T <: AbstractLRPRule} <: AbstractLRPRule
+    affine_rule::T
+end
+
+LayerNormRule() = LayerNormRule(ZeroRule())
+
+function lrp!(Rᵏ, rule::LayerNormRule, layer::LayerNorm, modified_layer, aᵏ, Rᵏ⁺¹)
+    # forward pass: split in normalization and scale
+    # aᵏ ->(normalize) aᵏₙ ->(scale) aᵏ⁺¹
+    ## normalize
+    eps = convert(float(eltype(aᵏ)), layer.ϵ)
+    μ = mean(aᵏ; dims = 1:length(layer.size))
+    σ = std(aᵏ, dims=1:length(layer.size), mean=μ, corrected=false)
```

Comment:
Can we introduce a variable for `1:length(layer.size)` since it gets used three times?

Reply:
done :)